mitodl/ocw-studio
# make markdown editor minimal by default

#### Pre-Flight checklist


#### What are the relevant tickets?
#1277 

#### What's this PR do?
1. Updates the frontend code to use `minimal=true` the default for markdown widget.
2. Syncs the studio configs with https://github.com/mitodl/ocw-hugo-projects/pull/182 _doing this manually was no fun and it seems like maybe we haven't done it recently. I think we've talked about just pulling in the configs for running tests? I will start a discussion_

#### How should this be manually tested?
1. Use the course config from https://github.com/mitodl/ocw-hugo-projects/pull/182
2. View any widget markdown widget for which `minimal` is not specified. A good example is Resource description. Verify that it is minimal (i.e., no tables, no code blocks, etc.)
3. Check that widgets for which minimal=false are still full, e.g., page bodies or resource bodies.
